tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -297,18 +343,47 @@
     "\n",
     "\n",
     "def get_representative_dataset(n_iter):\n",
+    "    \"\"\"\n",
+    "    This function creates a representative dataset generator\n",
+    "    Args:\n",
+    "        n_iter: number of iterations for MCT to calibrate on\n",
+    "\n",
+    "    Returns:\n",
+    "        A representative dataset generator\n",
+    "\n",
+    "    \"\"\"\n",
     "\n",
     "    def representative_dataset():\n",
+    "        \"\"\"\n",
+    "        Creates a representative dataset generator from a PyTorch data loader, The generator yields numpy\n",
+    "        arrays of batches of shape: [Batch, H, W ,C]\n",
+    "        Returns:\n",
+    "            A representative dataset generator\n",
+    "\n",
+    "        \"\"\"\n",
     "        ds_iter = iter(loader)\n",
     "        for _ in range(n_iter):\n",
     "            t = next(ds_iter)[0]\n",
-    "            yield [t.detach().cpu().numpy().transpose((0, 2, 3, 1))]\n",
+    "            # Convert the Torch tensor from the data loader to a numpy array and transpose to the\n",
+    "            # right shape: [B, C, H, W] -> [B, H, W, C]\n",
+    "            tf_shaped_tensor = t.detach().cpu().numpy().transpose((0, 2, 3, 1))\n",
+    "            yield [tf_shaped_tensor]\n",
     "\n",
     "    return representative_dataset\n",
     "\n",
     "\n",
+    "# Set IMX500-v1 TPC\n",
+    "tpc = mct.get_target_platform_capabilities(\"tensorflow\", 'imx500', target_platform_version='v1')\n",
+    "# set weights compression target, so the quantized model will fit the IMX500 memory\n",
```

Comment:
Maybe change to "set weights memory size" because you're not defining it as weights compression in this example, so it can be confusing